### PR TITLE
Tag new buckets with Vanta tags

### PR DESCRIPTION
**Pull request by leezen**
I've set myself as the owner as Vanta expects an actual user

### CODE REVIEW

File: scripts/sync-and-test-bucket.sh [hunk 1]
```diff
@@ -49,6 +49,9 @@ aws_region="$(pulumi -C infrastructure config get 'aws:region')"
 # fail, causing this script to exit nonzero. In either case, it's okay to continue.
 aws s3 mb $destination_bucket_uri --region $aws_region || true
 
+# Tag the bucket with ownership information
```
Comment (cnunciato) [inline scripts/sync-and-test-bucket.sh:52]: Super-ridiculous complete-sentence nit, sorry -- just something I picked up from others on the team and have come to believe in.  😄
 
```suggestion
# Tag the bucket with ownership information.
```

File: scripts/sync-and-test-bucket.sh [hunk 2]
```diff
@@ -49,6 +49,9 @@ aws_region="$(pulumi -C infrastructure config get 'aws:region')"
...
+aws s3api put-bucket-tagging --bucket $destination_bucket_uri --tagging file://bucket-tagging.json
```
Comment (cnunciato) [inline scripts/sync-and-test-bucket.sh:53]: I think this should be relative to the project root:

```suggestion
aws s3api put-bucket-tagging --bucket $destination_bucket_uri --tagging "file://$(pwd)/scripts/bucket-tagging.json"
```

**Comment by leezen** [comment]
Thanks for the suggestions. Yes, you're right that this should only apply to the production stack. These tags would be harmless for a dev stack, but also pretty unnecessary and if someone outside of the org wanted to try it, would be pretty nonsensical, so let me throw in a check for that.

File: scripts/sync-and-test-bucket.sh (hunk 2 above)
Comment (leezen) [reply scripts/sync-and-test-bucket.sh:53]: Good idea on just having an absolute path

**Comment by cnunciato** [comment]
🤔 Shouldn't `pulumi-docs-origin-pr-4036-c6dc43ad` have been tagged? It doesn't look like it was.

I think maybe [this line](https://github.com/pulumi/docs/pull/4036/files#diff-eb8fc496386a9f1c6146a288a305c02aR53) didn't work -- should it maybe be "pulumi/production"?

File: scripts/sync-and-test-bucket.sh [hunk 3]
```diff
@@ -49,6 +49,11 @@ aws_region="$(pulumi -C infrastructure config get 'aws:region')"
 # fail, causing this script to exit nonzero. In either case, it's okay to continue.
 aws s3 mb $destination_bucket_uri --region $aws_region || true
 
+# Tag the bucket with ownership information for production buckets.
+if [ "$(pulumi stack --show-name)" == "production" ]; then
```
Comment (cnunciato) [inline scripts/sync-and-test-bucket.sh:53]: Looking at the log, I think this should be:

```suggestion
if [ "$(pulumi -C infrastructure stack --show-name)" == "production" ]; then
```

**Comment by leezen** [comment]
Ah... I missed that last comment. You are totally right.

**Comment by leezen** [comment]
Fixing in https://github.com/pulumi/docs/pull/4045

**Pull request merged**